app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
include SessionsHelper dùng để làm gì em nhỉ?

Reply:
Dạ “include SessionsHelper” có nghĩa là import module Sessions Helper ở trong sessions_helper.rb (được tạo ra khi tạo Sessions controller).
Include SessionsHelper để sử dụng được các phương thức được định nghĩa trong module SessionsHelper. (VD: log-in() method).
Khi thêm module vào "application_controller" tự động được bao gồm trong “views”

Reply:
thế anh dùng extend thay cho include có được không? Khác gì nhau không?

Reply:
Không dùng extend thay thế include được ạ. Bởi vì: 
- Include gọi được các phương thức trong module SessionsHelper (coi như các phương thức của object) 
- Còn extend xem các phương thức trong modlue như là các hàm của class, được khai báo trong class để gọi ra.  
class A
   extend SessionsHelper 
end
=> gọi đc các phương thức trong SessionsHelper